web/client/api/MapConfigDAO.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2015, GeoSolutions Sas.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+var axios = require('axios');
+
+/**
+ * API for local config
+ */
+var Api = {
```

Comment:
It is a skeleton for configuration DAO. Can be used for remote configuration managing:
* errors
* headers
* authentication